js/model.js

```diff
@@ -622,30 +656,37 @@ helper.mixer = (function (mixerList) {
         return retVal;
     };
 
-    publicScope.loadServoRules = function (mixer) {
+    publicScope.loadServoRules = function (mixer, heliSettings) {
         SERVO_RULES.flush();
-
+        var fPitchRollWeight = parseFloat(heliSettings.pitchRollWeight)/100.0;
         for (const i in mixer.servoMixer) {
             if (mixer.servoMixer.hasOwnProperty(i)) {
                 const r = mixer.servoMixer[i];
+                var rate = r.getRate();
+                if ((mixer.platform==PLATFORM_HELICOPTER)&&(r.getInput()==INPUT_STABILIZED_YAW)&&heliSettings.useTailMotor)
+                    continue;
+                if ((mixer.platform==PLATFORM_HELICOPTER)&&((r.getInput()==INPUT_STABILIZED_ROLL)||(r.getInput()==INPUT_STABILIZED_PITCH)))
+                    rate *= fPitchRollWeight;
                 SERVO_RULES.put(
                     new ServoMixRule(
                         r.getTarget(),
                         r.getInput(),
-                        r.getRate(),
+                        Math.round(rate),
                         r.getSpeed()
                     )
                 );
             }
         }
     }
 
-    publicScope.loadMotorRules = function (mixer) {
+    publicScope.loadMotorRules = function (mixer, heliSettings) {
         MOTOR_RULES.flush();
 
         for (const i in mixer.motorMixer) {
             if (mixer.motorMixer.hasOwnProperty(i)) {
                 const r = mixer.motorMixer[i];
+                if ((mixer.platform==PLATFORM_HELICOPTER)&&(r.getYaw()!=0)&&!heliSettings.useTailMotor)
```

Comment:
Please use brackets even for single line conditions, it greatly increases readability

Reply:
OK. I'll fix it

Reply:
Fixed